pkg/cmd/server/api/types.go

```diff
@@ -980,3 +983,19 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// PodLimitRequestConfig is the configuration for the PodLimitRequest
+// admission controller which overrides user-provided container request/limit values.
+type PodLimitRequestConfig struct {
```

Comment:
I have no idea what future resource policy might look like, other than knowing that soon we'll be setting request/limit on the pod directly rather than containers. I could maybe see the scope of this plugin being expanded if other resources come into play? Not sure.

I went with `ClusterResourceOverride` and `ClusterResourceOverrideConfig`. Lemme know if you have any strong feelings about that.